app/sections/TransactionHistory/TransactionHistory.tsx

```diff
@@ -54,100 +59,135 @@ const TransactionHistory = () => {
     } catch (e) {
       console.log(e);
     }
-  }, [provider, depositContract, wallet, providerL2]);
+  }, [provider, depositContract, walletAddress, providerL2]);
 
   useEffect(() => {
     fetchTransactions();
   }, [fetchTransactions]);
 
   return (
     <Wrapper>
-      <h1 style={{ marginBottom: "5px" }}>Transaction History</h1>
-      <div>
-        Please note it could take 5-10 mins until you see the ETH on your L2
-        wallet.
-      </div>
+      <TitleText>Transaction History</TitleText>
+      {!walletAddress && (
+        <ConnectText>
+          Connect your wallet to view your transaction history.
+        </ConnectText>
+      )}
+      <TableRow>
+        <div>Address</div>
+        <div>Confirmations</div>
+        <div>Date</div>
```

Comment:
Merge Date and Time columns, put them first position, then Confirmation and Tx Link